src/models/state/wallet/wallet_configuration.rs

```diff
@@ -74,6 +74,12 @@ impl WalletConfiguration {
             }
         };
 
+        if cli_args.compose {
+            if let Some(scan_mode_configuration) = self.scan_mode.as_mut() {
+                scan_mode_configuration.set_guesser_fraction(cli_args.guesser_fraction);
+            };
+        }
+
```

Comment:
Composers are changing their guesser fraction often. I don't know how meaningful or helpful this parameter definition is here.